wizard.py

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
```

Comment:
Ah, the wizard script. @Zogg wrote it as an experiment. We were thinking that the wizard could ask the user a bunch of questions, then spit out the `docker run` command that will get mantl started for them. I don't know if we will actually include this for 1.1, it really depends upon if we want to go this direction.